common/src/main/resources/db/changelog/v001/AB2D-356-rename-columns.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE job RENAME job_id TO job_uuid;
```

Comment:
There is a unique constraint on the `job` table on `job_id` named `uc_job_job_id` both to enforce uniqueness and also for the index it creates.  I think you may need to recreate it to use the `job_uuid` column.

Reply:
@adaykin Let's re-create (or rename, if possible) the constraint and this should be good to go into `master`